app/src/loginService.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+    angular
+        .module('webshopModule')
+        .service('loginService', LoginService);
+
+    function LoginService(api, $window) {
+
+        var session = { authenticated: false };
+
+        this.login = function (email, password) {
+            return api.login(email, password)
+                .then(function (token) {
+                    session.authenticated = true;
+                    $window.localStorage.setItem('authenticated', true);
```

Comment:
If we use the LocalStorage, let's store the token in the session, and save the whole session. The `api` service will have to access the token.